pkg/kv/kvserver/protectedts/protectedts.go

```diff
@@ -139,6 +141,30 @@ type Verifier interface {
 	Verify(context.Context, uuid.UUID) error
 }
 
+// Metrics encapsulates the metrics exported by the reconciler.
+type Metrics struct {
+	ReconcilationRuns    *metric.Counter
+	RecordsProcessed     *metric.Counter
+	RecordsRemoved       *metric.Counter
+	ReconciliationErrors *metric.Counter
+}
+
+var _ metric.Struct = (*Metrics)(nil)
+
+// MetricStruct makes Metrics a metric.Struct.
+func (m *Metrics) MetricStruct() {}
```

Comment:
This part doesn't need to move, the server still holds on to the concrete struct.

Reply:
Yup, moved it back and exported the constructor instead.